packages/vue/index.js

```diff
@@ -18,6 +18,15 @@ module.exports = (neutrino, opts = {}) => {
     }
   }, opts);
 
+  // Add vue extension as a higher priority than JS files.
```

Comment:
I think we also need a change inside the `.has('lint')` block in this file, that updates the `test` regex to add `.vue` (since the lint preset will have already configured it, by the time options.extensions is updated here).